matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/DefaultSession.kt

```diff
@@ -113,6 +114,7 @@ internal class DefaultSession @Inject constructor(
         private val accountService: Lazy<AccountService>,
         private val defaultIdentityService: DefaultIdentityService,
         private val integrationManagerService: IntegrationManagerService,
+        private val thirdPartyService: ThirdPartyService,
```

Comment:
Is there a reason to not put it Lazy  and use a getter to namespace the functionnalityN

Reply:
No reason, I switch to your proposal.